src/rectangle_selection.jl

```diff
@@ -1,7 +1,12 @@
-# Returns true if a handle is clicked
-function is_clicked(pt::XY, handle::Handle, d)
-    return (handle.x - d/2 < pt.x < handle.x + d/2) &&
-        (handle.y - d/2 < pt.y < handle.y + d/2)
+# Returns the handle near click or an empty handle
+function nearby_handle(pt::XY, rh::RectHandle)
```

Comment:
A bit funny to have this do almost the same thing as `near_vertex`, but not quite. Still, this is fine.